src/azul/azulclient.py

```diff
@@ -104,6 +102,7 @@ def synthesize_notification(self,
             'source': {
                 'id': bundle_fqid.source.id,
                 'spec': str(bundle_fqid.source.spec),
+                'prefix': str(bundle_fqid.source.spec.prefix)
```

Comment:
Why do we need this? Isn't the prefix included in `str(bundle_fqid.source.spec)`?

Reply:
Removed.